src/App.jsx

```diff
@@ -1,35 +1,20 @@
 import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import Header from './Components/Header'
+import { Routes, Route } from 'react-router-dom';
+import NavBar from './Components/NavBar';
 import './App.css'
+import ArticleList from './Components/ArticleList';
 
 function App() {
-  const [count, setCount] = useState(0)
-
   return (
     <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+      <NavBar />
+      <Routes>
+        <Route path='/' element={<Header/>}/>
+        <Route path='/articles' element={<ArticleList/>}/>
```

Comment:
You may want it this way but if you want the header to always be visible, it shouldn't be in your routes as this conditionally renders components depending on the url. :)